components/brave_wallet/common/eth_sign_typed_data_helper.cc

```diff
@@ -241,9 +241,9 @@ absl::optional<std::vector<uint8_t>> EthSignTypedDataHelper::EncodeField(
     }
   } else if (base::StartsWith(type, "uint", base::CompareCase::SENSITIVE)) {
     // uint8 to uint256 in steps of 8
-    unsigned type_check;
-    if (!base::StringToUint(type.data() + 4, &type_check) || type_check % 8 ||
-        type_check > 256)
+    unsigned num_bits;
+    if (!base::StringToUint(type.data() + 4, &num_bits) || num_bits % 8 ||
```

Comment:
can we use `!ValidSolidityBits(num_bits)` here?
also here
https://github.com/brave/brave-core/pull/12958/files#diff-b42b2e3b6e17d197b286511a6aeedd8bd63033c8b515126cbb6dad235e903deeR277

Reply:
Rebased into b451cd524e607a2d856c3e274e79678398feeba0